sdk/communication/test-resources.json

```diff
@@ -46,9 +72,21 @@
     }
   ],
   "outputs": {
+    "AZURE_TENANT_ID": {
```

Comment:
export the AZURE_SUBSCRIPTION_ID env var here

Reply:
Added to outputs